core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1876,13 +1876,19 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
 
     for {
       googleProjectId <- traceWithParent("getGoogleProjectFromBuffer", span)(_ => resourceBufferService.getGoogleProjectFromBuffer(projectPoolType, workspaceId))
+      _ = logger.info(s"Moving google project ${googleProjectId} to folder.")
```

Comment:
We decided to leave our log messages in, because people kept needing to add them, so we decided they were useful enough to stay.